### ISSUE TITLE
[NBug] Value must not be null.

### ISSUE THREAD

**Issue by RussKie** [gitextensions/gitextensions#8951]
Looks like a regression introduced in 2450e8b5

## Steps to reproduce

1. Clone c7d7be502cfd8092e9bda28a614fac5d6bb1ad9a
2. `.\build.cmd -c Release -launch`
3. Open "Settings"

## Error Details
```
System.Exception: Value must not be null.
   at Microsoft.Validates.NotNull[T](T value)
   at GitUIPluginInterfaces.PseudoSetting.CreateControlBinding()
   at GitUI.CommandsDialogs.SettingsDialog.Plugins.PluginSettingsPage.CreateSettingsControls()
   at GitUI.CommandsDialogs.SettingsDialog.Plugins.PluginSettingsPage.Init(IGitPlugin gitPlugin)
   at GitUI.CommandsDialogs.FormSettings.<OnRuntimeLoad>b__18_1(IGitPlugin plugin)
   at System.Linq.Enumerable.WhereSelectListIterator`2.MoveNext()
   at System.Linq.Buffer`1..ctor(IEnumerable`1 source)
   at System.Linq.OrderedEnumerable`1.<GetEnumerator>d__1.MoveNext()
   at GitUI.CommandsDialogs.FormSettings.OnRuntimeLoad(EventArgs e)
   at GitUI.GitExtensionsForm.OnLoad(EventArgs e)
   at System.Windows.Forms.Form.OnCreateControl()
   at System.Windows.Forms.Control.CreateControl(Boolean fIgnoreVisible)
   at System.Windows.Forms.Control.CreateControl()
   at System.Windows.Forms.Control.WmShowWindow(Message& m)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.ScrollableControl.WndProc(Message& m)
   at System.Windows.Forms.Form.WmShowWindow(Message& m)
   at System.Windows.Forms.Form.WndProc(Message& m)
   at ResourceManager.GitExtensionsFormBase.WndProc(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
```

## Additional information
Tried to open "Settings" when this happened

## Environment
- Git Extensions 33.33.33
- Build c7d7be502cfd8092e9bda28a614fac5d6bb1ad9a
- Git 2.30.0.windows.2
- Microsoft Windows NT 10.0.19041.0
- .NET Framework 4.8.4300.0
- DPI 96dpi (no scaling)

**Comment by RussKie** [comment]
Looks like this may the fix:
```diff
diff --git a/Plugins/GitUIPluginInterfaces/Settings/PseudoSetting.cs b/Plugins/GitUIPluginInterfaces/Settings/PseudoSetting.cs
index 41d932a9d..a2ace7637 100644
--- a/Plugins/GitUIPluginInterfaces/Settings/PseudoSetting.cs
+++ b/Plugins/GitUIPluginInterfaces/Settings/PseudoSetting.cs
@@ -44,14 +44,14 @@ public PseudoSetting(string text, string caption = "    ", int? height = null,
 
         public ISettingControlBinding CreateControlBinding()
         {
-            Validates.NotNull(_textBoxCreator);
             return new PseudoBinding(this, CustomControl, _textBoxCreator);
         }
 
         private class PseudoBinding : SettingControlBinding<PseudoSetting, Control>
         {
-            private readonly Func<TextBox> _textBoxCreator;
-            public PseudoBinding(PseudoSetting setting, Control? customControl, Func<TextBox> textBoxCreator)
+            private readonly Func<TextBox>? _textBoxCreator;
+
+            public PseudoBinding(PseudoSetting setting, Control? customControl, Func<TextBox>? textBoxCreator)
                 : base(setting, customControl)
             {
                 _textBoxCreator = textBoxCreator;
@@ -59,6 +59,7 @@ public PseudoBinding(PseudoSetting setting, Control? customControl, Func<TextBox
 
             public override Control CreateControl()
             {
+                Validates.NotNull(_textBoxCreator);
                 Setting.CustomControl = _textBoxCreator();
                 return Setting.CustomControl;
             }
```

**Comment by mstv** [comment]
Fixed by #8949.

**Comment by RussKie** [comment]
Omg I committed wrong files to the wrong PR..... :(((
